frontend/internal-packages/agent/src/db-agent/createDbAgentGraph.ts

```diff
@@ -0,0 +1,42 @@
+import { END, START, StateGraph } from '@langchain/langgraph'
+import { designSchemaNode } from '../chat/workflow/nodes/designSchemaNode'
+import { createAnnotations } from '../chat/workflow/shared/langGraphUtils'
```

Comment:
I think these should be moved to `agent/src/db-agent/*`. I would like to group the directories by agent.
This will be addressed in the next PR.